### PR TITLE
fix sse HttpClient

### PR DESCRIPTION
**Pull request by Raffiki**
- fix `scala.MatchError`
- exchange `if else` body

goes along with https://github.com/Spinoco/protocol/pull/51

### CODE REVIEW

File: src/main/scala/spinoco/fs2/http/HttpClient.scala [hunk 1]
```diff
@@ -148,10 +148,13 @@ trait HttpClient[F[_]] {
 
       def sse[A : SSEDecoder](rq: HttpRequest[F], maxResponseHeaderSize: Int, chunkSize: Int): Stream[F, A] =
         request(rq, chunkSize, maxResponseHeaderSize, Duration.Inf).flatMap { resp =>
-          if (resp.header.headers.exists { case `Content-Type`(ct) => ct.mediaType == MediaType.`text/event-stream`  })
-            Stream.raiseError(new Throwable(s"Received response is not SSE: $resp"))
-          else
+          if (resp.header.headers.exists { _ match {
```
Comment (mraulim) [inline src/main/scala/spinoco/fs2/http/HttpClient.scala]: You don't need the `_ match {` part.

**Comment by mraulim** [comment]
@Raffiki Thank you!

**Comment by Raffiki** [comment]
Hi @mraulim , is there an upcoming release planned? if not, I will need to fork/publish

**Comment by Raffiki** [comment]
@mraulim @pchlupacek any update on a release? This would make the sse endpoint actually working. Also, this goes hand in hand with a release of spinico/protocol, so a version upstep of this dependency is also required

**Comment by AdamChlupacek** [comment]
@Raffiki sorry, this completely went under my radar, I am releasing a version with your fix now. Please do not be afraid to hit us on gitter next time.

**Comment by AdamChlupacek** [comment]
@Raffiki as for the protocols, I have something else I want to get in there within a week or so, so that will have to wait a bit, sorry about that :/

**Pull request merged**